student-test/assignment1/Test_validate_username.c

```diff
@@ -14,9 +14,8 @@
 */
 void test_validate_my_username()
 {
-    /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
-     * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     char expected[20] , actual[20]; // declaring strings to compare
+     strcpy(expected, my_username()); 
+      strcpy(actual, malloc_username_from_conf_file()); 
```

Comment:
@aysvarya-gopinath what happens to the string returned by `malloc_username_from_conf_file` here?